frontend/src/app/components/modals/create-namespace-bucket-modal/create-namespace-bucket-modal.js

```diff
@@ -250,7 +251,8 @@ class CreateNamespaceBucketModalViewModel extends ConnectableViewModel {
             closeModal(),
             createNamespaceBucket(bucketName, readPolicy, writePolicy,
                 this.enableCaching() ? {
-                    ttl_ms: cacheTTL
+                    ttl_ms: cacheTTL,
+                    eviction: eviction_type
```

Comment:
Because this is a constant, I think it is better to encode it directly in the epic that is sending the request 
at `frontend/src/app/epics/create-namespace-bucket.js.
There is no need to pass the information through the ViewModel.

Reply:
irrelevant in latest changes